_licenses/ncsa.txt

```diff
@@ -0,0 +1,65 @@
+---
+title: University of Illinois/NCSA Open Source License
+spdx-id: NCSA
+nickname: UIUC/NCSA
+source: https://opensource.org/licenses/NCSA
+
+description: The University of Illinois/NCSA Open Source License, or UIUC license, is a permissive free software license, based on the <a href="/licenses/mit/">MIT/X11 license</a>  and the <a href="/licenses/bsd-3-clause/">BSD 3-clause License</a>. By combining parts of these two licenses, it attempts to be clearer and more concise than either. It's conditions include requiring the preservation of copyright and license notices both in source and in binary distributions and the prohibtion of using the names of the authors or the project organization to promote or endorse derived products.
+
+how: Create a text file (typically named LICENSE or LICENSE.txt) in the root of your source code and copy the text of the license into the file. Replace [year] with the current year and [fullname] with the name (or names) of the copyright holders. Replace [project] with the project organization, if any, that sponsors this work.
+
+using: 
+ - Clang: https://github.com/llvm-mirror/clang/blob/master/LICENSE.TXT 
+ - LLVM: https://github.com/llvm-mirror/llvm/blob/master/LICENSE.TXT
+ - LLDB: https://github.com/llvm-mirror/lldb/blob/master/LICENSE.TXT
+
+permissions:
+  - commercial-use
+  - modifications
+  - distribution
+  - private-use
+
+conditions:
+  - include-copyright
+
+limitations:
+  - liability
+  - warranty
+
+---
+
+University of Illinois/NCSA Open Source License 
+
+Copyright (c) [year], [fullname] 
+All rights reserved.
+
+Developed by: [fullname]
+              [project]
+              [email]
```

Comment:
I guess `email` is best that can be done with fields available, but a URL is clearly the practice here for this license. I wonder if we should/could make `project_url` or similar available (the github.com interface would probably substitute with the repo URL)? cc @benbalter